src/main/scala/org/broadinstitute/hail/driver/Write.scala

```diff
@@ -22,7 +25,13 @@ object Write extends Command {
   def requiresVDS = true
 
   def run(state: State, options: Options): State = {
-    state.hadoopConf.delete(options.output, true)
+    if (options.overwrite)
+      state.hadoopConf.delete(options.output, recursive =  true)
+    else if (state.hadoopConf.exists(options.output))
+      fatal(
+        s"""File already exists at ${options.output}
+           |  Choose this path or rerun with --overwrite flag,
+           |  but do NOT overwrite a VDS being read.""".stripMargin)
```

Comment:
I suspect this line will be confusing to our users. What exactly are you trying to prevent?